lib/benchfella/cli_util.ex

```diff
@@ -17,12 +17,10 @@ defmodule Benchfella.CLI.Util do
   def locate_snapshots(count \\ 2) do
     dir = "bench/snapshots"
     snapshots =
-      case File.ls(dir) do
-        {:error, _}  -> []
-        {:ok, files} ->
-          files |> Enum.sort |> Enum.reverse |> Enum.take(count) |> Enum.reverse
-      end
-      |> Enum.map(&Path.join(dir, &1))
+      Path.join(dir, "*.snapshot")
+      |> Path.wildcard
+      |> Enum.take(-count)
```

Comment:
As I see neither `:filelib.wildcard` nor `prim_file:list_dir` does sorting, I think it is up to the OS, e.g. mtime or whatnot. Will add `Enum.sort` to be on safe side.